src/main/kotlin/day12/Function.kt

```
@@ -0,0 +1,41 @@
+}
+
+
+class MyComparator : Comparator<Int> {
+    override fun compare(o1: Int?, o2: Int?): Int {
+        return o1!! - o2!!
+    }
+}
+
+fun command(func: Attackable) {
+
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)